fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -106,9 +106,9 @@ private void exportBinary(final URI uri)
             writeResponse(response, file);
         }
     }
-    private void exportContainer(final URI uri)
+    private void exportDescription(final URI uri, final URI... describes)
             throws FcrepoOperationFailedException, IOException {
-        final File file = fileForContainer(uri);
+        final File file = fileForContainer(describes.length > 0 ? describes[0] : uri);
```

Comment:
Though I don't think we can support multiple descriptions if we save them like the RDF serializer does — only one can have the same name as the binary.  So I think this would mean naming them based on the repository URI instead (e.g., `.../rest/file1/fcr:metadata.ttl` instead of `.../rest/file1.ttl`.